docs/subnets/network-upgrade.md

```diff
@@ -0,0 +1,138 @@
+# Network Upgrade
```

Comment:
it's a bit confusing to call this a network upgrade here and a subnet upgrade in the other document. Can we change this document to be the WAGMI Upgrade instead?